### PR TITLE
Nextcloud io

### PR DESCRIPTION
**Pull request by akristen**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+<figcaption>
+After setting up the Nextcloud integration with New Relic, your data displays in dashboards like these right out of the box. 
+</figcaption>
+
+## Install the infrastructure agent [#infra]
+
+To get your data into New Relic, install our infrastructure agent. Our infrastructure agent collects and ingests data from your app so you can keep track of your app’s performance.
+
+You can install the infrastructure agent a few different ways.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:26]:
```suggestion
You can install the infrastructure agent a few different ways: 
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+After setting up the Nextcloud integration with New Relic, your data displays in dashboards like these right out of the box. 
+</figcaption>
+
+## Install the infrastructure agent [#infra]
+
+To get your data into New Relic, install our infrastructure agent. Our infrastructure agent collects and ingests data from your app so you can keep track of your app’s performance.
+
+You can install the infrastructure agent a few different ways. 
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs both the infrastructure agent and the best application monitoring agent for your system. To learn more about how our guided install works, check out our [overview doc](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
```
Comment (a-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:28]:
```suggestion
* (Guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd): Guided install is a CLI tool that inspects your system and installs both the infrastructure agent and the best application monitoring agent for your system. To learn more about how our guided install works, check out our [overview doc](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview). 
* Manual install: If you’d rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](), [Windows](), or [iOS](). 
```
^Missing links for manual install

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+</figcaption>
+
+## Install the infrastructure agent [#infra]
+
+To get your data into New Relic, install our infrastructure agent. Our infrastructure agent collects and ingests data from your app so you can keep track of your app’s performance.
+
+You can install the infrastructure agent a few different ways. 
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs both the infrastructure agent and the best application monitoring agent for your system. To learn more about how our guided install works, check out our [overview doc](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview). 
+* If you’d rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](), [Windows](), or [iOS]().
```
Comment (a-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:29]:
```suggestion
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+3. Clone `nextcloud-exporter`:
+
+```
+git clone https://github.com/xperimental/nextcloud-exporter.git
+cd nextcloud-exporter
+```
+
+4. Create a binary file in the `nextcloud-exporter` folder by using `make`. and Name the file `nextcloud-exporter`.
+5. Complete token authentication. You need to be using Nextcloud 22 or later.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:51]: Consider referencing this at the top as a requirement

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+TOKEN=$(openssl rand -hex 32)
+# Set token using the occ command by going to "/var/www/nextcloud”
+occ config:app:set serverinfo token --value "$TOKEN"
+``` 
+6. From your cloned Nextcloud folder, run the following command to initiate username and password authentication:
+
+```
+./nextcloud-exporter --login <USERNAME> <PASSWORD> --server http://127.0.0.1
+```
+7. To access metrics for Nextcloud, run:
```
Comment (a-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:64]:
```suggestion
7. To access metrics for Nextcloud, run this command:
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+```
+./nextcloud-exporter -u <YOUR_USERNAME> -p <YOUR_PASSWORD> --server http://127.0.0.1:9205
+```
+
+From your browser, go to `http:127.0.0.1:9205/metrics` to view your metrics.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:70]:
```suggestion
8. From your browser, go to `http:127.0.0.1:9205/metrics` to view your metrics.  
```

**Review by a-sassman — APPROVED** [review]
LGTM other than a few open items.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx (hunk 3 above)
Comment (akristen) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx]: Fixed!

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+1. Run the following command to change your directory: `cd /etc/newrelic-infra/integrations.d/`
+2. Create a Prometheus configuration file: `sudo nano nri-prometheus-config.yml`
+3. Add that Prometheus OpenMetrics configuration file to `nri-prometheus-config.yml`. 
+4. To get server-related information, replace the URL inside the targets:
+
+```yml
+targets:
+  urls: ["http://127.0.0.1:9205"]
+```
+5. Restart your New Relic infrastructure service by running `sudo systemctl restart newrelic-infra.service`
```
Comment (homelessbirds) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:85]: Do you need a period at the end of this sentece?

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+4. Create a binary file in the `nextcloud-exporter` folder by using `make`. and Name the file `nextcloud-exporter`.
+5. Complete token authentication. You need to be using Nextcloud 22 or later.
+
+```
+# Generate random value using OpenSSL.
+TOKEN=$(openssl rand -hex 32)
+# Set token using the occ command by going to "/var/www/nextcloud”
+occ config:app:set serverinfo token --value "$TOKEN"
+``` 
+6. From your cloned Nextcloud folder, run the following command to initiate username and password authentication:
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:59]: 8. From your cloned Nextcloud folder, run the following command to start and nextcloud using username and password authentication:

NOTE: kindly correct my sentence formation if wrong. 👍

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+```
+./nextcloud-exporter -u <YOUR_USERNAME> -p <YOUR_PASSWORD> --server http://127.0.0.1:9205
+```
+
+8. From your browser, go to `http:127.0.0.1:9205/metrics` to view your metrics.  
+
+## Integrate Prometheus OpenMetrics with Nextcloud [#prometheus]
+
+With your Nextcloud metrics exposed, you can use our Prometheus OpenMetrics integration to scrape data.
+
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:75]: Install our [Prometheus OpenMetrics integration](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-update-or-uninstall-your-prometheus-openmetrics-integration/)

You need to configure the nri-prometheus-config.yml file. Your configuration file should match our snippet in the [nri-prometheus](https://github.com/newrelic/nri-prometheus/blob/main/configs/nri-prometheus-config.yml.sample) repository

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+1. Run the following command to change your directory: `cd /etc/newrelic-infra/integrations.d/`
+2. Create a Prometheus configuration file: `sudo nano nri-prometheus-config.yml`
+3. Add that Prometheus OpenMetrics configuration file to `nri-prometheus-config.yml`.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:78]: the above points 1, 2 and 3 can be removed

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+8. From your browser, go to `http:127.0.0.1:9205/metrics` to view your metrics.  
+
+## Integrate Prometheus OpenMetrics with Nextcloud [#prometheus]
+
+With your Nextcloud metrics exposed, you can use our Prometheus OpenMetrics integration to scrape data.
+
+1. Run the following command to change your directory: `cd /etc/newrelic-infra/integrations.d/`
+2. Create a Prometheus configuration file: `sudo nano nri-prometheus-config.yml`
+3. Add that Prometheus OpenMetrics configuration file to `nri-prometheus-config.yml`. 
+4. To get server-related information, replace the URL inside the targets:
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:79]: To send nextcloud metrics, replace the URL inside the targets with the URL to which the metrics are exposed. Here metrics are exposed on "http://127.0.0.1:9205":

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx (hunk 7 above)
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:85]: [Restart your New Relic infrastructure service](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/)

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Review by homelessbirds — APPROVED** [review]
Looks good!

**Pull request merged**